docs/contracts/deployed-contracts.md

```diff
@@ -0,0 +1,11 @@
+---
+title: ⛓ Deployed Contracts
```

Comment:
Can you add metadata too? A description would be nice. You can see the syntax on other pages

https://docusaurus.io/docs/markdown-features/head-metadata

I think page description would be nice.